frontend/src/v5/store/projects/projects.selectors.ts

```diff
@@ -45,5 +45,5 @@ export const selectCurrentProjectName = createSelector(
 
 export const selectIsProjectAdmin = createSelector(
 	selectCurrentProjectDetails,
-	(project): boolean => project?.isAdmin || null,
+	(project): boolean => project?.isAdmin ?? null,
```

Comment:
Why this change? in any case I would return false instead of null as this selector returns a boolean in theory